assets/stylesheets/forms/_inputs.scss

```diff
@@ -0,0 +1,275 @@
+// Form Elements
+//
+// Weight: 6
+//
+// Style guide: FormElements
+
+// Inputs
+//
+// Markup:
+//  <div class="form-field">
+//   <label for="text">Test</label>
+//   <input type="text" placeholder="Placeholder" id="text" />
+//  </div>
+//  <div class="form-field">
+//   <label for="text" class="disabled">Test</label>
+//   <input type="text" placeholder=":disabled" disabled="disabled" id="text" />
+//  </div>
+//  <div class="form-field--has-errors">
+//   <label for="text">Test</label>
+//   <input aria-invalid="true" type="text" value="text" id="text" />
+//   <span>Something is wrong.</span>
+//  </div>
+//
+// Weight: 1
+//
+// Style guide: FormElements.Inputs
+
+$border-color: #29384A;
+$border-color-shadow: black;
+$placeholder-color: #888FA8;
+$focus-color: #2278F1;
+$focus-color-shadow: #CBE6FF;
+
+@mixin inputStyle{
+    display: block;
+    border-radius: 2px;
+    font-size: 14px;
+    height: 40px;
+    color: color('carbon');
+    padding: 0 15px; 
+    color: color(carbon);  
+    margin-bottom: 10px; 
+    line-height: 100%;
+    outline: none;
+
+    ::-webkit-input-placeholder{
```

Comment:
I would think that this is not necessary in out desktop and mobile version as we use https://github.com/kimroen/ember-cli-autoprefixer there. Maybe we should implement it into the styleguide here as well?

(but not really necessary as we use it in the Rails project also, just an idea to make it consistent)